examples/3d-tiles/tile-3d-layer.js

```diff
@@ -316,23 +314,12 @@ export default class Tile3DLayer extends CompositeLayer {
     let parsedColors = colors;
 
     if (isRGB565) {
-      parsedColors = new Uint8ClampedArray(pointsCount * 4);
+      parsedColors = new Uint8ClampedArray(pointsCount * 3);
       for (let i = 0; i < pointsCount; i++) {
         const color = parseRGB565(colors[i]);
         parsedColors[i * 4] = color[0];
         parsedColors[i * 4 + 1] = color[1];
         parsedColors[i * 4 + 2] = color[2];
-        parsedColors[i * 4 + 3] = 255;
-      }
-    }
-
-    if (colors && colors.length === pointsCount * 3) {
```

Comment:
Does cutting this remove support for RGB point clouds?

Reply:
it is still supported by specifying the size when passing to layer attributes